decoder/src/log/json_logger.rs

```diff
@@ -64,15 +61,31 @@ impl JsonLogger {
     }
 }
 
-#[derive(Serialize)]
-struct Path {
+#[derive(Debug, Deserialize, Serialize)]
+pub struct Json {
+    data: String,
+    host_timestamp: DateTime<Utc>,
+    level: String,
+    path: Option<Path>,
+    target_timestamp: String,
+
+    // backtrace is omitted from output if it is `None`
+    #[serde(skip_serializing_if = "Option::is_none")]
+    backtrace: Option<Vec<()>>,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct Path {
```

Comment:
if this now includes the module path and the source code location maybe call it `Location` -- here and in the Json field.

Reply:
The problem is that we already have a public `defmt_decoder::elf2table::Location`. I will try to come up with better names.

Reply:
Renamed it to Location anyways.